ephios/plugins/simpleresource/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from ephios.core.models import Shift
+
+
+class ResourceCategory(models.Model):
+    name = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.name
```

Comment:
pylint complains: E0307: __str__ does not return str (invalid-str-returned)
@felixrindt should i just annotate it to ignore it?

Reply:
annotate it